xbmc/interfaces/legacy/InfoTagMusic.h

```diff
@@ -103,6 +103,30 @@ namespace XBMCAddon
 #ifdef DOXYGEN_SHOULD_USE_THIS
       ///
       /// \ingroup python_InfoTagMusic
+      /// @brief \python_func{ getMediaType() }
+      ///-----------------------------------------------------------------------
+      /// Get the media type of the music item.
+      ///
+      /// @return [string] media type
+      ///
+      /// Available strings about media type for music:
+      /// | String         | Description                                       |
+      /// |---------------:|:--------------------------------------------------|
+      /// | artist         | To define it as an artist
```

Comment:
we have these docs for video mediatype as well:
https://github.com/xbmc/xbmc/blob/master/xbmc/interfaces/legacy/InfoTagVideo.h#L253-L268

if addons are to interpreted to return value, i don't think it's bad to list the possible values.